cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -69,13 +70,31 @@ type NetConf struct {
 	PluginLogLevel string `json:"pluginLogLevel"`
 }
 
+// Implements encoding.TextUnmarshaler from comma-separated string.
+type IPList []net.IP
+
+func (l *IPList) UnmarshalText(data []byte) error {
```

Comment:
Used by [LoadArgs](https://github.com/containernetworking/cni/blob/master/pkg/types/args.go#L101) when parsing arguments to the CNI through reflection, so never called directly.